### DISCUSSION TITLE
Issues are going to slavarazum/laravel-wave repo

### DISCUSSION THREAD

**Discussion by leandrodiogenes** [qruto/laravel-wave-client#3] [Q&A] [answered]
Those links are redirecting to another repository.
![image](https://user-images.githubusercontent.com/2463299/183539350-8f75b006-cdd9-4936-8d0c-d98a258b951f.png)

**Comment by leandrodiogenes** [comment]
Sorry, opened 2 times.

**Comment by slavarazum** [comment, marked as answer]
Thank you. Corrected the links.
